README.md

```diff
@@ -76,4 +77,16 @@ By default, installing a package with the `—save` option, npm saves the packag
 npm config set save-exact
 ```
 
+[↑ back to Table of Contents](#table-of-contents)
+
+## Write atomic tasks
+
```

Comment:
I would try to explain what an "atomic task" is in a very short summary, as it is a new vocabulary which we introduce.